content/articles/icann-domain-validation.markdown

```diff
@@ -0,0 +1,46 @@
+---
+title: ICANN domain validation requirements
+excerpt: This support article explains what ICANN domain validation is and how to ensure that your domain name is not suspended by ICANN due to non-validation.
+categories:
+  - Domains
+---
+
+# ICANN domain validation
+
+ICANN, the organization that oversees domain name registrations for most top-level domains, requires validation of your registrant email address whenever a new domain is registered, or your registrant email address or name is changed. Failure to validate your registrant email address results in suspension of the domain name after 15 days of non-compliance.
+
+When you make a change to your registrant's email address or name, you will receive an email similar to the following:
+
+![ICANN Verification Email](/files/icann-verification-email.png)
+
+This email will be sent from a `@dnsimple.com` email address and will include a link similar to the following:
+
+`http://raa.name-services.com/raaverification/verification.aspx?VerificationCode=A8E3763E-EE70-42DB-A654-20BF560300A00`
```

Comment:
I already changed the text... I can change it further if you want (to all zeros for example).